velox/common/memory/ByteStream.cpp

```diff
@@ -18,6 +18,11 @@
 
 namespace facebook::velox {
 
+std::string ByteRange::toString() const {
+  return fmt::format(
+      "ByteRange[size[{}] pos[{}]]", succinctBytes(size), position);
```

Comment:
nit: too many []

perhaps, ByteRange[10MB starting at 45]

Also, do we need to include "ByteRange"?